### ISSUE TITLE
parallel fit

### ISSUE THREAD

**Issue by xgdgsc** [xiaodaigh/JLBoost.jl#51]
Does it support [Acceleration and Parallelism · MLJ](https://alan-turing-institute.github.io/MLJ.jl/v0.7/acceleration_and_parallelism/#) ?

**Comment by xiaodaigh** [comment]
No. I am currently focused on writing a parquet writer so once I am done with that I intend to return to JLBoost to update it.


I see the feature is experimental at the moment.
